libcontainer/console_linux.go

```diff
@@ -89,7 +89,14 @@ func (c *linuxConsole) mount(rootfs, mountLabel string, uid, gid int) error {
 	if f != nil {
 		f.Close()
 	}
-	return syscall.Mount(c.slavePath, dest, "bind", syscall.MS_BIND, "")
+	if err := syscall.Mount(c.slavePath, dest, "bind", syscall.MS_BIND, ""); err != nil {
+		return err
+	}
+
+	if err := syscall.Mount("", dest, "", syscall.MS_PRIVATE, ""); err != nil {
+		return err
```

Comment:
@lk4d4 You agree with putting more verbose error information? I can do that. But this is just one place. We are changing mount properties at many places. And I can't see why these are any different.

For that matter why any other syscall return error is any different. There is always scope to add more information around it. But we don't do it. Or we leave it to author and see if it makes sense to add more info around it.

In this case this is just one instance where mount private failed. There are many more so I don't feel strongly about adding this extra message.